web/src/reactions.js

```diff
@@ -1,4 +1,5 @@
 import $ from "jquery";
+import assert from "minimalistic-assert";
```

Comment:
Unused import in this commit; looks like next commit removes this.

Reply:
Oops, fixed.